### PR TITLE
Added remainder function and remainder feature tested and works

### PR DESCRIPTION
**Pull request by TreySchneider95**
Pull request for the remainder feature.

### CODE REVIEW

File: calculator.py [hunk 1]
```diff
@@ -4,13 +4,20 @@
...
+def remainder_func(a,b):
+	return a % b
+
 if __name__ == '__main__':
 	parser = argparse.ArgumentParser()
 
 	#add arguments here
-	parser.add_argument('--add',mnargs=2, type=int, help='adds two numbers')
+	parser.add_argument('--add',nargs=2, type=int, help='adds two numbers')
+	parser.add_argument('--remainder',nargs=2, type=int, help='adds two numbers')
```
Comment (dd-code-immersives) [inline calculator.py:15]: update help message for remainder to be accurate

Comment (TreySchneider95) [reply calculator.py]: Updated help message to be accurate

**Review by dd-code-immersives — APPROVED** [review]

**Comment by dd-code-immersives** [comment]
good work

**Review by dd-code-immersives — CHANGES_REQUESTED** [review]
add handling for when the denominator is 0

**Review by dd-code-immersives — APPROVED** [review]

**Pull request merged**